api/orchestrator/orchestrator.proto

```diff
@@ -949,6 +963,17 @@ message RemoveCertificateRequest {
   string certificate_id = 1 [ (validate.rules).string.min_len = 1 ];
 }
 
+message Metadata {
+  string clouditor_version = 1;
+  string golang_version = 2;
+  repeated Dependency dependency = 3; 
```

Comment:
dependencies?

Reply:
Ah, you meant plural instead of singular?

Reply:
Yes